Functions/RIPA.Functions.Submission/Services/REST/StopService.cs

```diff
@@ -145,7 +145,7 @@ public  Listperson_Stopped CastToDojListPersonStopped(RIPA.Functions.Common.Mode
                         },
                         Gend = CastToDojPercievedGender(personStopped.PerceivedGender),
                         LGBT = personStopped.PerceivedLgbt ? "Y" : "N",
-                        GendNC = personStopped.GenderNonconforming ? "5" : string.Empty
+                        GendNC = (bool)personStopped.GenderNonconforming ? "5" : string.Empty
```

Comment:
Modify this statement to check if value is null, true, or false and set the value based on that.